webroot/results/includes/statistics/most_solves_in_one_competition_or_year.php

```diff
@@ -39,6 +39,26 @@
   LIMIT     50
 " );
 
+$allTime = polishMostSolvesAttempts( "
+  SELECT    personId,
+            "swag" whereId,
+            count(value1>0 or null)+
+            count(value2>0 or null)+
+            count(value3>0 or null)+
+            count(value4>0 or null)+
+            count(value5>0 or null) solves,
+            count(value1 and value1<>-2 or null)+
+            count(value2 and value2<>-2 or null)+
+            count(value3 and value3<>-2 or null)+
+            count(value4 and value4<>-2 or null)+
+            count(value5 and value5<>-2 or null) attempts
+  FROM      Results, Competitions competition
+  WHERE     competition.id = competitionId
```

Comment:
I'm surprised to see a join on Competitions here. Is this some reason for this that I'm missing?